benchmarks/src/bench.rs

```diff
@@ -57,6 +62,38 @@ pub static TARGET_ITERATION_COUNT: Lazy<usize> = Lazy::new(|| {
         .expect("invalid TARGET_ITERATION_COUNT")
 });
 
+static SMALL_DOC: OnceCell<Document> = OnceCell::const_new();
+pub async fn get_small_doc() -> Document {
+    SMALL_DOC
+        .get_or_init(|| async {
+            let data_path = DATA_PATH
+                .join("single_and_multi_document")
+                .join("small_doc.json");
+            let mut file = spawn_blocking_and_await!(std::fs::File::open(data_path))
```

Comment:
Nit: It might be worth factoring out a utility function like
```
async fn from_json_file<T: DeserializeOwned>(path: impl AsRef<Path>) -> Result<T>
```